### PR TITLE
added 30-Seconds-Of-C++

### PR DESCRIPTION
**Pull request by Bhupesh-V**

### CODE REVIEW

File: _data/projects/30SecondsOfC++.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: 30 Seconds of C++
+desc: A collection of STL features (functions/libraries) of C++ which can be learned in 30 seconds or less.
+site: https://github.com/Bhupesh-V/30-seconds-of-cpp
+tags:
+- c++
+- c++11
+- c++14
+- c++17
+upforgrabs:
+  name: 30-Seconds-of-C++
```
Comment (ritwik12) [inline _data/projects/30SecondsOfC++.yml]: @Bhupesh-V This should be the link to your label, such as "good first issue", "up-for-grabs" etc. and also the following link should be for that only. Please change

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/30SecondsOfC++.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+desc: A collection of STL features (functions/libraries) of C++ which can be learned in 30 seconds or less.
+site: https://github.com/Bhupesh-V/30-seconds-of-cpp
+tags:
+- c++
+- c++11
+- c++14
+- c++17
+upforgrabs:
+  name: good first contribution
+  link: https://github.com/Bhupesh-V/30-seconds-of-cpp/issues
```
Comment (ritwik12) [inline _data/projects/30SecondsOfC++.yml]: @Bhupesh-V You don't see to have a label by the name of "good first contribution", please create a new label for this name or use labels from herehttps://github.com/Bhupesh-V/30-seconds-of-cpp/labels. Also, please add the label link for eg: https://github.com/Bhupesh-V/30-seconds-of-cpp/labels/good%20first%20issue

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/30SecondsOfC++.yml (hunk 2 above)
Comment (Bhupesh-V) [reply _data/projects/30SecondsOfC++.yml]: Ok 
Can I add multiple issue labels or it's just limited to only one ??

Comment (ritwik12) [reply _data/projects/30SecondsOfC++.yml]: Only one

Comment (ritwik12) [reply _data/projects/30SecondsOfC++.yml]: @Bhupesh-V

Comment (Bhupesh-V) [reply _data/projects/30SecondsOfC++.yml]: Does it need any other changes ?? @ritwik12

File: _data/projects/30SecondsOfC++.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+site: https://github.com/Bhupesh-V/30-seconds-of-cpp
+tags:
+- c++
+- c++11
+- c++14
+- c++17
+- stl
+upforgrabs:
+  name: beginner-friendly 
+  link: https://github.com/Bhupesh-V/30-seconds-of-cpp/issues?q=is%3Aopen+is%3Aissue+label%3Abeginner-friendly
```
Comment (ritwik12) [inline _data/projects/30SecondsOfC++.yml]: @Bhupesh-V Please correct this to https://github.com/Bhupesh-V/30-seconds-of-cpp/labels/beginner-friendly

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/30SecondsOfC++.yml (hunk 2 above)
Comment (ritwik12) [reply _data/projects/30SecondsOfC++.yml]: Yep, change the link please, https://github.com/Bhupesh-V/30-seconds-of-cpp/labels/beginner-friendly

File: _data/projects/30SecondsOfC++.yml (hunk 3 above)
Comment (Bhupesh-V) [reply _data/projects/30SecondsOfC++.yml]: Done :+1:

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@Bhupesh-V Thanks!!

**Pull request merged**